Helper/Cart.php

```diff
@@ -1962,7 +1962,7 @@ public function collectDiscounts(
                         case RuleInterface::COUPON_TYPE_SPECIFIC_COUPON:
                         case RuleInterface::COUPON_TYPE_AUTO:
                             $couponCode = $quote->getCouponCode();
-                            $description = trim(__('Discount ') . $rule->getDescription());
+                            $description = trim(__('Discount ') . $rule->getDescription(). " ({$quote->getCouponCode()})");
```

Comment:
```suggestion
                            $description = trim(__('Discount ') . $rule->getDescription(). " ($couponCode)");
```

Reply:
oops good call, missed this when merging conflict.